state/execution_test.go

```diff
@@ -99,6 +101,31 @@ func TestBeginBlockValidators(t *testing.T) {
 	}
 }
 
+func (state State) createBlock(
```

Comment:
why not use state.MakeBlock?

Reply:
state.MakeBlock expects state.LastValidators not to be empty because of the way how BFT time is set. This is the point discussed on Slack. But this test forces empty validator set (I don't really understand what is being tested).

Reply:
I will check